.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - nightly
```

Comment:
Not sure 5.5 is worth the efffort... its EOL is quite soon.It only has < 2 months of security patches.